src/transaction/log_manager.c

```diff
@@ -839,7 +839,7 @@ log_create_internal (THREAD_ENTRY * thread_p, const char *db_fullname, const cha
    * Turn off creation bits for group and others
    */
 
-  (void) umask (S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH);
+  (void) umask (S_IRGRP | S_IWGRP | S_IXGRP | S_IROTH | S_IWOTH | S_IXOTH);
```

Comment:
from `continuous-integration/appveyor/pr`, S_IXGRP and S_IXOTH doesn't seem to be defined on Windows.

Reply:
I verified this comments and I cannot understand why this error arises. 
At first, I will try to "push" git branch to test the only "umask (S_IRGRP)".

Because I think that this error can arises by additional bit permissions.